Tests/ParameterSpaces/parameter_space_mock.hpp

```diff
@@ -95,6 +95,7 @@ class A2dParameterSpaceMock : public sources::parameter_spaces::ParameterSpace<2
 
  private:
   void NurbsBookExe3_8KnotVectorsDegrees();
+  void NurbsBookExe3_8InsertRemoveKnotDegreeBasis();
   void NurbsBookExe3_8Bezier();
   void NurbsBookExe3_8BezierElevateReduce();
   void NurbsBookExe3_8BezierElevatedTwice();
```

Comment:
Please rename `NurbsBookExe3_8InsertRemoveKnotDegreeBasis` `NurbsBookExe3_8InsertRemoveKnotVectorsDegreesBasisFunctions`; afterwards sort alphabetically and ensure that the order in the corresponding `.inc` file matches this alphabetical order.

Reply:
Which `.inc` file do you mean?

Reply:
Sorry, my bad! Since we are dealing with the mock here, I mean the corresponding `.cpp` file of course: `parameter_space_mock.cpp`.

Reply:
I see! I now updated the private methods to be alphabetically sorted at the end of the `.cpp`.

Reply:
Thanks! Once you renamed `NurbsBookExe3_8InsertRemoveKnotDegreeBasisFunctions` `NurbsBookExe3_8InsertRemoveKnotVectorsDegreesBasisFunctions` I will be able to resolve this conversation :)